model/clusters_mgmt/v1/backup_state_value_type.model

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright (c) 2025 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// BackupStateValue represents the state of a backup
+enum BackupStateValue {
```

Comment:
@miguelsorianod had an interesting point when we last discussed about the enryption at host feature (enabled/disabled): if we use enum, and we later expand it, it's considered an API breaking change on the client side. However, if we use string instead (although they are represented the same way in the json as enum), it's not considered a breaking change and will not necessary require client to be modified. 

Hence, I suggest to change it to be string and document the allowed values.

Reply:
Done.

Reply:
good catch, thanks!

 I forgot to delete the model file.